src/components/common/CirrusFavicon.tsx

```diff
@@ -53,7 +53,7 @@ interface CirrusFaviconProps {
   status?: BuildStatus | TaskStatus | boolean;
 }
 
-export default (props: CirrusFaviconProps) => {
+export default function CirrusFavicon(props: CirrusFaviconProps): JSX.Element {
```

Comment:
Was there an error here? Or is it some kind of automatic fix?

Reply:
This PR updated eslint, which introduced a new rule that suggests against using anonymous default exports, so I used next.js's codemod to fix it.

As for .yarn/plugins and .yarn/releases, the idea behind it is that every repository keeps it's own copy of the package manager.  It's like the gradle wrapper, but instead of a wrapper it's the whole thing.  It has to be committed to work.

Reply:
What do you think of using [non zero install](https://yarnpkg.com/getting-started/qa#which-files-should-be-gitignored). We do still download things from the internet so I guess there is not that much benefit of commiting these files and keeping them up to date.

Reply:
I've had long conversations about zero installs in the yarn discord, and I ultimately think using zero-installs is a bad idea.